src/main/java/com/google/devtools/build/lib/analysis/config/CoreOptionConverters.java

```diff
@@ -234,6 +234,42 @@ public String getTypeDescription() {
     }
   }
 
+  /**
+   * Flag converter for canonicalizing a label (possibly with a "/..." suffix) and/or define by
+   * converting the label to ambiguous canonical form.
```

Comment:
Do you mean "unambiguous canonical form", or is "ambiguous" correct here and I am misunderstanding?

Reply:
I really did mean "ambiguous" since the implementation used `toString()`, but I now changed this to `getUnambiguousCanonicalForm()` for clarity.